core/controllers/admin.py

```diff
@@ -656,6 +675,390 @@ def _generate_dummy_explorations(
         else:
             raise Exception('Cannot generate dummy explorations in production.')
 
+    def _create_dummy_interactions(self, state, dest):
+        """Creates a dummy interaction and links it to a destination.
+
+        Args:
+            state: state_domain.State. Description.
+            dest: state_domain.State. Description.
+        """
+        state.update_interaction_id('TextInput')
+        state.update_interaction_customization_args({
+            'placeholder': {
+                'value': {
+                    'content_id': 'ca_placeholder_0',
+                    'unicode_str': ''
+                }
+            },
+            'rows': {'value': 1}
+        })
+
+        state.update_next_content_id_index(1)
+        state.update_linked_skill_id(None)
+        state.update_content(
+            state_domain.SubtitledHtml('1', 'Content 1'))
+        recorded_voiceovers = state_domain.RecordedVoiceovers({})
+        written_translations = state_domain.WrittenTranslations({})
+        recorded_voiceovers.add_content_id_for_voiceover(
+            'ca_placeholder_0')
+        recorded_voiceovers.add_content_id_for_voiceover('1')
+        recorded_voiceovers.add_content_id_for_voiceover(
+            'default_outcome')
```

Comment:
I think it'd be cleaner if we grouped the property initialization and update together, e.g.:

```
create record voiceovers
update record voicovers

create written translations
update written translations

...
```

That way the object initialization is kept close to the site where its used.